kibana-reports/server/executor/createScheduledReport.ts

```diff
@@ -33,33 +34,36 @@ export const createScheduledReport = async (
   esReportsClient: ILegacyClusterClient,
   notificationClient: ILegacyClusterClient,
   logger: Logger
-): Promise<CreateReportResultType> => {
+) => {
   const isScheduledTask = true;
   let createReportResult: CreateReportResultType;
 
   const {
-    report_definition: { report_params: reportParams },
+    report_definition: {
+      report_params: reportParams,
+      delivery: { delivery_type: deliveryType },
+    },
   } = report;
   const { report_source: reportSource } = reportParams;
 
-  // compose url with localhost
-  const completeQueryUrl = `${LOCAL_HOST}${report.query_url}`;
   try {
-    // generate report
-    if (reportSource === REPORT_TYPE.savedSearch) {
-      createReportResult = await createSavedSearchReport(
-        report,
-        esClient,
-        isScheduledTask
-      );
-    } else {
-      // report source can only be one of [saved search, visualization, dashboard]
-      createReportResult = await createVisualReport(
-        reportParams,
-        completeQueryUrl,
-        logger
-      );
-    }
+    // TODO: generate report logic will be added back once we have the user impersonation
+    // if (reportSource === REPORT_TYPE.savedSearch) {
```

Comment:
Do we need to leave this commented-out code in?

Reply:
Yes I leave it on purpose, once we have user impersonation added, this code snippet will be reused.